hazelcast-sql/src/test/java/com/hazelcast/sql/impl/expression/predicate/BetweenOperatorIntegrationTest.java

```diff
@@ -165,14 +167,14 @@ public void betweenAsymmetricPredicateTypeCheckTest() {
 
                     Tuple2<List<SqlRow>, HazelcastSqlException> comparisonEquivalentResult = executePossiblyFailingQuery(
                             // the queries have extra spaces so that the errors are on the same positions
-                            "SELECT this FROM map WHERE this >=      ? AND this <= ?",
+                            "SELECT this FROM map WHERE this >=      ? AND this <= ?  ORDER BY this",
```

Comment:
Why `ORDER BY` is suddenly needed?

Reply:
Result set is not sorted by default, so it's easier to sort it during query execution and return a result set.

Reply:
But sorted result wasn't needed before, was it?

Reply:
Sorted result was produced before, yes. In case of Jet's normal scan, it doesn't guarantee that result set would be sorted.